sycl/include/sycl/functional.hpp

```diff
@@ -23,6 +23,7 @@ template <typename T = void> using bit_xor = std::bit_xor<T>;
 
 // std:logical_and/std::logical_or with a non-void type returns bool,
 // sycl requires returning T.
+// these definition should be consistent with std::max, std::min
```

Comment:
These definitions / This definition ?

Reply:
It meant "sycl::minimum and sycl::maximum" by these. I will fix the comment

Reply:
Why it's marked as resolved? There is no fix in this PR.